desc/transform.py

```diff
@@ -85,13 +85,14 @@ def __init__(
 
     def _set_up(self):
 
-        self.method = self._method
+        self.method = self._method  # assign according to logic in setter function
+        n = np.max(self.derivatives) + 1
```

Comment:
lower bound to 1

Reply:
Making this change passed the failing test. Should I make both changes. `n = np.maximum(np.amax(self.derivatives), 1) + 1`

Reply:
probably doesn't hurt, but I think `derivatives` should have a minimum value of 0

Reply:
Ok I'll revert that then